src/ensembl/src/shared/components/layout/StandardAppLayout.test.tsx

```diff
@@ -82,6 +82,17 @@ describe('StandardAppLayout', () => {
       ).toBe(1);
     });
 
+    it.only('main content remains full width when sidebar behaviour is slideover', () => {
```

Comment:
✂️ only

also, let's make the keyword agree grammatically with the description:
- `test('main area styles do not change when sidebar slides over it', () => {})`
or
- `it('does not change main area styles when sidebar slides over it', () => {})`

Reply:
Ouch, forgot to remove that.